apps/src/applab/commands.js

```diff
@@ -1381,7 +1384,8 @@ applabCommands.startWebRequest = function (opts) {
   apiValidateType(opts, 'startWebRequest', 'callback', opts.func, 'function');
   var req = new XMLHttpRequest();
   req.onreadystatechange = applabCommands.onHttpRequestEvent.bind(req, opts);
-  req.open('GET', opts.url, true);
+  var url = XHR_PROXY + encodeURIComponent(opts.url.replace(' ', '%20'));
```

Comment:
It's not a global replacement - you're only replacing the first space.  To do global replace, pass a regex with the `g` flag.

```javascript
opts.url.replace(/ /g, '%20')
```